src/pages/Explore/Explore.jsx

```diff
@@ -3,8 +3,14 @@ import "./Explore.css";
 import Navbar from "../../components/Navbar/Navbar";
 import Sidebar from "../../components/Sidebar/Sidebar";
 import VideoCard from "../../components/VideoCard/VideoCard";
+import { useVideoContext } from "../../context/video-context";
 
 const Explore = () => {
+  const { videoList } = useVideoContext();
+  console.log(videoList);
```

Comment:
Take care to cleanup consoles before pushing for deployment

Reply:
Sure!